README.md

```diff
@@ -1101,8 +1102,35 @@ terragrunt = {
 If you run `terragrunt apply-all --terragrunt-source /source/infrastructure-modules`, then the local path Terragrunt
 will compute for the module above will be `/source/infrastructure-modules//networking/vpc`.
 
+#### Skipping modules
 
+Consider the following file structure:
+
+```
+root
+├── terraform.tfvars
+├── prod
+│   └── terraform.tfvars
+├── dev
+│   └── terraform.tfvars
+└── qa
+    └── terraform.tfvars
+```
 
+In some cases, the root level terraform.tfvars is simply used to DRY up your terraform configuration and is simply
+included by the other terraform.tfvars files. In this case, you do not want the `xxx-all` commands to process the root
+level terraform.tfvars since it does not define any infrastructure by itself. To make the `xxx-all` commands skip the
+root level terraform.tfvars file, you can set skip = true in the terragrunt { ... } block as shown in this example:
+
+```hcl
+terragrunt = {
+  skip = true
```

Comment:
Will this `skip` only with `xxx-all` commands? Or is it a more general-purpose feature that skips `all` Terragrunt commands (e.g., to prevent a module from being modified unless you really really mean it)? I'm tempted to say the latter is the way to go, so perhaps these docs should be right next to those of `prevent_destroy`.